### ISSUE TITLE
I can use "torch.softmax(...)" in my codes but I couldn't find it in docs.

### ISSUE THREAD

**Issue by MichaelWei7** [pytorch/pytorch#19460]
## ❓ Questions and Help

When I search "softmax" or "torch.softmax" in pytorch docs, the related result is 
 "torch.nn.functional.softmax (...)". But I can use "torch.softmax(dim =1 )" in my codes. So, what happened? Pytorch version is 1.0.1.
By the way, how can I use softmax function outside the model? What's the best way?

**Comment by soumith** [comment]
this should not be used. see torch.nn.functional.softmax and there are docs for that.

**Comment by soumith** [comment]
(it will be removed soon)

**Comment by MichaelWei7** [comment]
> (it will be removed soon)

Thank you!
